iOS Client/Store/MBPodcastsStore.swift

```diff
@@ -27,7 +27,7 @@ class MBPodcastsStore {
     }
     
     func syncPodcasts() -> Promise<[Podcast]> {
-        let streams: [MBClient.PodcastStream] = [.pz, .mockingPulpit, .mockingCast]
+        let streams: [PodcastStream] = [.pz, .mockingPulpit, .mockingCast]
```

Comment:
We could pass these streams in a parameter to achieve one source of truth perhaps.